addons/stock_account/models/stock_move.py

```diff
@@ -550,6 +550,7 @@ def _prepare_account_move_vals(self, credit_account_id, debit_account_id, journa
             'stock_valuation_layer_ids': [(6, None, [svl_id])],
             'move_type': 'entry',
             'is_storno': self.env.context.get('is_returned') and self.env.company.account_storno,
+            'company_id': self.env.company.id,
```

Comment:
```suggestion
            'company_id': self.company_id.id,
```
This would be better actually

Reply:
Are you sure? We do exactly `self.env.company` the. line above, and would it be the same in any case?

Reply:
Damn, good catch
Can you change that one too ?